src/reader_utils.cc

```diff
@@ -67,6 +67,32 @@ Result<std::pair<IdType, IdType>> GetAdjListOffsetOfVertex(
                         static_cast<IdType>(array->Value(1)));
 }
 
+Result<IdType> GetVertexChunkNum(const std::string& prefix,
```

Comment:
Is the function used for getting the number of vertex chunks, or the number of actual gar files? Maybe reading vertices number from file and counting the chunk num is better, since there may be no properties attached.

Reply:
Good suggestions，will use vertices number to get the chunk num